### PR TITLE
samples: cdc_acm: add README.rst and use Arduino 101 as default board

### PR DESCRIPTION
**Pull request by jfischer-no**
Add README.rst and use Arduino 101 as default board

### CODE REVIEW

File: samples/subsys/usb/cdc_acm/README.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Requirements
+************
+
+This project requires an USB device driver,
+which is available for Intel Quark* SE SoC based boards like
+:ref:`arduino_101`.
+For the Arduino 101 board you need a USB to 3.3V TTL serial cable.
+Connect it to UART1 as described in
+:ref:`Connecting Serial Output section <arduino_101>`.
```
Comment (dbkinder) [inline samples/subsys/usb/cdc_acm/README.rst]: instead:

     the Connecting Serial Output section of :ref:`arduino_101`.

File: samples/subsys/usb/cdc_acm/README.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+Building and Running
+********************
+
+Arduino 101
+===========
+
+To see the consoles outputs of the app, open a serial port emulator and
```
Comment (dbkinder) [inline samples/subsys/usb/cdc_acm/README.rst]: s/consoles/console/      s/outputs/output/

File: samples/subsys/usb/cdc_acm/README.rst [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+   $ cd samples/subsys/usb/cdc_acm
+   $ make BOARD=arduino_101 flash
+
+See :ref:`Flashing the x86 Application Core section <arduino_101>` for details.
+
+Running
+=======
+
+You may need to stop modemmanager, if it is trying to access the
```
Comment (dbkinder) [inline samples/subsys/usb/cdc_acm/README.rst]: How will they know if this is a problem?  How do they stop modemmanager?

File: samples/subsys/usb/cdc_acm/README.rst [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+   usb 9-1: Product: CDC-ACM
+   usb 9-1: Manufacturer: Intel
+   usb 9-1: SerialNumber: 00.01
+   cdc_acm 9-1:1.0: ttyACM1: USB ACM device
+
+The app prints on serial output (UART1), used for the console:
+
+.. code-block:: console
+
+   Wait for DTR
```
Comment (dbkinder) [inline samples/subsys/usb/cdc_acm/README.rst]: Since minicom isn't running yet, I won't see this "Wait for DTR" output, right?

**Review by dbkinder — CHANGES_REQUESTED** [review]
edits suggested

File: samples/subsys/usb/cdc_acm/README.rst (hunk 4 above)
Comment (jfischer-phytec-iot) [inline samples/subsys/usb/cdc_acm/README.rst]: You would see, it will be printed on zephyr console and not via CDC ACM device.

Comment (dbkinder) [inline samples/subsys/usb/cdc_acm/README.rst]: Ah, so I'd need a **second** minicom session running to see that output.  That explains my confusion.

**Review by dbkinder — DISMISSED** [review]

**Review by nashif — APPROVED** [review]

**Pull request merged**